conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/GramineEntryPoint.kt

```diff
@@ -38,29 +71,71 @@ object GramineEntryPoint {
         return port
     }
 
+    /**
+     * Load enclave metadata properties file.
+     * This file contains properties required to load the enclave, such as the signing key measurement, threading level,
+     * and whether to use gramine-sgx or gramine-direct.
```

Comment:
The later point of gramine-sgx vs gramine-direct is not relevant here and my confuse someone later looking at this code.